jumptrading/aminal
# Add setting for maximum number of lines in the terminal buffer

## Description

Now the maximum number of lines in terminal buffers are limited via configuration file (`max_lines` parameter). By default, the `max_lines` parameter is set to 1000. 

Fixes #10

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

* Run Aminal
* In the Aminal window, run `dir /s` (this will produce more than 1000 lines of output).
* Scroll back the Aminal window and see that it contains no more than 1000 lines
* Run Task Manager and note the Aminal's memory consumption
* In the Aminal window, run `dir /s` again
* Note that the memory consumption has not been changed

**Test Configuration**:
* OS: `any supported`
* OS version: `n/a`
* Go version: `go1.11.2`

## Checklist:
